nvdaHelper/UIARemote/remoteLog.h

```diff
@@ -14,6 +14,14 @@ This license can be found at:
 
 #pragma once
 
+// Converts a utf8 encoded string into a utf16 encoded wstring
+std::wstring stringToWstring(const std::string& from) {
+	int wideLen = MultiByteToWideChar(CP_UTF8, 0, from.c_str(), from.length(), nullptr, 0);
+	std::vector<wchar_t> wideBuf (wideLen + 1, L'\0');
+	MultiByteToWideChar(CP_UTF8, 0, from.c_str(), from.length(), wideBuf.data(), wideLen);
+	return std::wstring(wideBuf.data());

```

Comment:
Looks like C++ standard library has you covered here. No need to use the Windows C API: https://stackoverflow.com/a/38383389

``` C++
#include <iostream>
#include <string>
#include <locale>
#include <codecvt>
std::string str = "your string in utf8";
std::wstring_convert<std::codecvt_utf8_utf16<char16_t>> converter;
std::wstring wstr = converter.from_bytes(str);
```


For reference, since C++17 `basic_string` returns a non-const pointer via `data()` you can use it as a buffer for string types to save a copy. In this case you don't need to explicitly reserve space for the trailing null char.
```suggestion
	std::wstring wideBuf (wideLen, L'\0');
	MultiByteToWideChar(CP_UTF8, 0, from.c_str(), from.length(), wideBuf.data(), wideLen);
	return wideBuf;

```